tests/unit/customizations/test_utils.py

```diff
@@ -43,23 +43,49 @@ def test_rename_command_table(self):
 class TestCommandTableAlias(BaseAWSHelpOutputTest):
 
     def test_alias_command_table(self):
-        handler = lambda command_table, **kwargs: utils.alias_command(
-            command_table, 'ec2', 'foo')
+        old_name = 'ec2'
+        new_name = 'nopossiblewaythisisalreadythere'
+
+        def handler(command_table, **kwargs):
+            utils.alias_command(command_table, old_name, new_name)
+
+        self._assert_commands_exist(old_name, new_name, handler)
+
+        # Verify that the new name is documented
+        self.driver.main(['help'])
+        self.assert_contains(new_name)
+        self.assert_not_contains(old_name)
+
+    def test_hide_new_command_name(self):
+        old_name = 'ec2'
+        new_name = 'nopossiblewaythisisalreadythere'
+
+        def handler(command_table, **kwargs):
+            utils.alias_command(command_table, old_name, new_name, False)
+
+        self._assert_commands_exist(old_name, new_name, handler)
+
+        # Verify that the new isn't documented
+        self.driver.main(['help'])
+        self.assert_not_contains(new_name)
+        self.assert_contains(old_name)
+
+    def _assert_commands_exist(self, old_name, new_name, handler):
```

Comment:
Can we break up this general ``_assert_commands_exist`` into several other ``_assert_*()`` functions? It seems to be a bit more than checking if it is just existing. If I wanted to check if it existed, I would just run ``help`` on the command and not necessarily all of the other help commands. In the end it will just make it a bit easier to read when going through the test.